pallets/fragments/src/weights.rs

```diff
@@ -86,8 +86,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
 	// Storage: Fragments FragmentsListSize (r:1 w:1)
 	// Storage: Fragments FragmentsList (r:0 w:1)
-	fn upload() -> Weight {
-		(160_000_000 as Weight)
+	fn upload(l: u32, ) -> Weight {
+		(157_800_000 as Weight)
+			// Standard Error: 163_000
+			.saturating_add((1_600_000 as Weight).saturating_mul(l as Weight))
```

Comment:
we are making progress but those numbers are HUGE, how can we tweak those?

Reply:
We can do this as a part of separate issue